docs/user-guide/volumes.md

```diff
@@ -108,6 +108,10 @@ While tmpfs is very fast, be aware that unlike disks, tmpfs is cleared on
 machine reboot and any files you write will count against your container's
 memory limit.
 
+For monitoring and reference purposes: The default location where `emptyDir` 
```

Comment:
I don't really think we should document this - it's really part of the implementation.  Consider kubelet in chroot or container - this will be wrong.

@vishh @dchen1107